plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/CodeTemplatesTest.java

```diff
@@ -237,11 +240,50 @@ private Element validatePom()
         new DefaultArtifactVersion(pluginVersion.getTextContent());
     DefaultArtifactVersion expected = new DefaultArtifactVersion("1.3.2");
     Assert.assertTrue(artifactVersion.compareTo(expected) >= 0);
+    
+    XPath xpath = XPathFactory.newInstance().newXPath();
+    xpath.setNamespaceContext(new Maven4NamespaceContext());
+    NodeList dependencyManagementNodes = (NodeList) xpath.evaluate(
+        "./m:dependencyManagement",
+        root,
+        XPathConstants.NODESET);
+    Assert.assertEquals(1, dependencyManagementNodes.getLength());
+    
+    String bomGroupId = (String) xpath.evaluate(
```

Comment:
Worth checking that there is a single `<dependencies>` element?

Reply:
No, we're not. We're getting the first top-level dependencies element.